lib/pbench/test/unit/server/query_apis/test_index_mappings.py

```diff
@@ -0,0 +1,190 @@
+import pytest
+from http import HTTPStatus
+from sqlalchemy.exc import DatabaseError
+
+from pbench.server.database.models.template import Template
+
+
+@pytest.fixture()
+def run_template(monkeypatch, fake_mtime):
```

Comment:
Did you really need to duplicate the `find_template` fixture instead of just expanding it to include the full template body?

I'd generally prefer to avoid having multiple fixtures for "the same thing", especially because they're both mocking the run index entry.